conda_smithy/templates/azure-pipelines-linux.yml.tmpl

```diff
@@ -15,7 +15,7 @@ jobs:
       {{ config_name }}:
         CONFIG: {{ config_name }}
         UPLOAD_PACKAGES: {{ upload }}
-        DOCKER_IMAGE: {{ config["docker_image"][-1] }}
+        DOCKER_IMAGE: {{ config["docker_image"][0] }}
```

Comment:
Can you revert this change to reduce the diff?

Reply:
Sure, done.